mdpow/fep.py

```diff
@@ -1384,14 +1394,15 @@ def p_transfer(G1, G2, **kwargs):
             logger.info("The solvent is %s .", G.solvent_type)
             logger.info("Estimator is %s.", estimator)
             logger.info("Free energy calculation method is %s.", G.method)
-            if estimator == 'mdpow':
-                G.analyze(**G_kwargs)
-            elif estimator == 'alchemlyb':
-                if G_kwargs['SI']:
-                    logger.info("Statistical inefficiency analysis will be performed.")
-                else:
-                    logger.info("Statistical inefficiency analysis won't be performed.")
-                G.analyze_alchemlyb(**G_kwargs)
+
+        if estimator == 'mdpow':
+            G.analyze(**G_kwargs)
+        elif estimator == 'alchemlyb':
+            if G_kwargs['SI']:
+                logger.info("Statistical inefficiency analysis will be performed.")
+            else:
+                logger.info("Statistical inefficiency analysis won't be performed.")
+            G.analyze_alchemlyb(**G_kwargs)
```

Comment:
This whole block needs to remain indented under the `if kwargs['force'] or (not hasattr(G.results.DeltaA, 'Gibbs')):`. – @VOD555 correct me if I am wrong.